tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -2959,7 +2959,7 @@ namespace iter_ops {
             trace t{};
             I first{t};
             same_as<iter_difference_t<I>> auto const result = distance(move(first), default_sentinel);
-            STATIC_ASSERT(!noexcept(distance(move(first), default_sentinel))); // No conditional noexcept
+            STATIC_ASSERT(noexcept(distance(move(first), default_sentinel))); // No conditional noexcept

```

Comment:
I believe that the comment here should be dropped; we are strengthening both overloads with conditional `noexcept` now (instead of just the `sized_sentinel_for<decay_t<_It>>` case).